src/shared/components/views/DatatableWithToggle.tsx

```diff
@@ -57,18 +87,15 @@ const DatatableWithToggle: FC = ({ children }) => {
         >
           {children}
         </datatableElement.name>
-        {/* TODO show button only when the content exceeds visible area of the table  
-        {tableRef?.current?.rows.length > 7 && (
-          
+        {(showButton || expandTable) && (
```

Comment:
`|| expandTable` is because if the button has been triggered the scroll will disappear and `showButton` will become false, so make sure to ignore that here otherwise there would not be a way to collapse the table again.